vault/community.events.office-hours.md

```diff
@@ -2,25 +2,31 @@
 id: 1eec6c7b-242e-4102-93f1-fc285702ecd2
 title: Office Hours
 desc: ''
-updated: 1637801961738
+updated: 1641250731190
 created: 1619981785169
+skipLevels: 2
+has_collection: true
+sort_by: date
+sort_order: reverse
 ---
 
+## Summary
+
 We hold office hours in our [Discord Channel](https://discordapp.com/channels/717965437182410783/776450886230605845/837055285042348046). During Dendron Office Hours, the Dendron team highlights new features and open the floor to community Q&A in the Dendron Discord. Please ask questions and drop on by if you can!
 
 Office Hours are regularly recorded and uploaded to the [Office Hours YouTube Playlist](https://www.youtube.com/playlist?list=PLrXlRqY7c8J__ASIOrPQZn6DfHC-xxcO0).
 
 <!--
 iframe embed for active Luma event series
-Currently for 2021 Q4
+Currently for 2022 Q1
 -->
 <iframe
```

Comment:
the dimensions are wrong, doesn't span the page and is cropped

Reply:
Ah, I used the default iframe embed code that luma provides. I've extended the height, updated width to 100%. If attempting to mess with spanning in other ways, such as 100% height, it renders strangely since the iframe contents themselves will seem to dynamically adjust themselves. This looks to render best, and looks good in a mobile view and scales well as the page does.